### PR TITLE
Add contributors

### LINKED ISSUE #197: Should add `contributors` to package.json

### PR DESCRIPTION
**Pull request by hjdivad**
[fix #197]

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,18 @@
     "ember-data-addon"
   ],
   "license": "MIT",
-  "author": "",
+  "author": "David J. Hamilton <dhamilton@linkedin.com>",
+  "contributors": [
```
Comment (runspired) [inline package.json]: I'm unfamiliar with the limitations of this field, but are we able to add github / npm emails here?

Comment (rwjblue) [reply package.json]: Can be either of these formats:

```js
[
  { name: 'Robert Jackson', email: 'me@rwjblue.com', url: 'https://www.rwjblue.com' }
]
```

Or:

```js
[
  'Robert Jackson <me@rwjblue.com> (https://www.rwjblue.com)'
]
```

Any of the three parts can be left off.

Documentation here: https://docs.npmjs.com/files/package.json#people-fields-author-contributors

**Comment by hjdivad** [comment]
test failures are from changes in ember-data; it's not immediately clear but all of our stages except `test` run against latest b/c ember-try runs with `--no-lockfile`

File: package.json (hunk 1 above)
Comment (xg-wang) [reply package.json]: "Thomas Wang <wangxgwxg@gmail.com> (https://twitter.com/xinganwang)"

Thanks :)

Comment (scalvert) [reply package.json]: 'Steve Calvert <steve.calvert@gmail.com> (https://www.twitter.com/scalvert)'

Comment (dnachev) [reply package.json]: 'Danail Nachev (http://www.linkedin.com/in/danailnachev)'

Comment (sangm) [reply package.json]: "Sang Mercado sang.mercado@gmail.com (http://linkedin.com/in/sangmercado)"

Comment (dnalagatla) [reply package.json]: "Dinesh Nalagatla dinu.nalagatla@gmail (https://www.linkedin.com/in/dineshnalagatla)"
Thank you

Comment (hjdivad) [reply package.json]: > Hopefully, it's OK to skip on the email :) Not that all mail list doesn't have it already.

It's 💯 .  The entire reason I didn't preemptively include emails is I that I don't want to publish peoples' contact info without their explicit preference, even if it's ultimately a pretty futile gesture.

**Pull request merged**